apache/vhosts/standard.sls

```diff
@@ -6,9 +6,8 @@ include:
 {% for id, site in salt['pillar.get']('apache:sites', {}).items() %}
 {% set documentroot = site.get('DocumentRoot', '{0}/{1}'.format(apache.wwwdir, site.get('ServerName', id))) %}
 
-{{ id }}:
-  file:
-    - managed
+{{ id }}-vhosts:
```

Comment:
This is a good improvement, wdyt of making it `apache_vhosts_{{ id }}` or `apache-vhosts-{{ id }}` so the name of the formula is there to? (For the record, I'm OK with merging it even if you don't like the suggestion :smile: )

Reply:
I like the suggestion